packages/acs-ui-common/src/telemetry.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// TODO: Find a way to import version from ../package.json
+// This package is rooted at acs-ui-common/src so acs-ui-common/package.json is not accessible.
+const version = '0.0.0-alpha';
```

Comment:
Any ideas for reading version off of `package.json`

Doing this:

```
  import { version } from '../package.json`
```

doesn't work because the package is rooted at `acs-ui-common/src`. Importing from `acs-ui-common/package.json` messes with `tsc` (it ends up rooting the package at `acs-ui-common`.

Reply:
Probably will need a prebuild step for this that writes this to a `currentVersion.ts` or something similar that this file can export from, don't think tsc can do this.

I worry about `require()` any file that is not meant to be part of the output as the output code will keep that require line and hence the package.json will need to be in the right place in the packaged up dist folder.

Reply:
Alternatively you could use a package like this: https://github.com/indexzero/node-pkginfo
That crawls up the directory and uses the first package.json it finds - but not ideal

Reply:
Other option:
* hard code into a file
* create/overwrite a `rush bump` command
* have it perform `beachball bump` && `manual command to update hard coded value`
* Add unit test to ensure this hard coded value always matches the package.json

Reply:
Yeah, seems like must choose the "least bad". I was thinking of the build step approach. Better than any non-conventional setup at runtime as it quickly blows up due to the different ways our library gets used.